ecs.tf

```diff
@@ -273,6 +273,10 @@ resource "aws_ecs_task_definition" "service" {
             name : "WARPSTREAM_REGION",
             value : var.control_plane_region
           },
+          {
+            name : "WARPSTREAM_KAFKA_PORT"
+            value : "${var.kafka_port}"
```

Comment:
```suggestion
            name : "WARPSTREAM_KAFKA_PORT",
            value : tostring(var.kafka_port)
```

Reply:
the reason is due to the error

```
cannot unmarshal number into Go struct field KeyValuePair.Environment.Value of type string
```
which means that an environment variable is being assigned a number value, but ECS expects environment variable values to always be strings.

Reply:
Oh yes thanks! Was still working on this PR, will be fixing that :)